### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by RawatH**

### CODE REVIEW

File: .idea/other.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shubham1g5) [inline .idea/other.xml:1]: `.idea/` files should not be committed and should be added to `.gitignore` instead.

File: app/src/main/java/org/commcare/dalvik/abha/ui/main/fragment/AbhaValidationFragment.kt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package org.commcare.dalvik.abha.ui.main.fragment
+
+class AbhaValidationFragment {
```
Comment (shubham1g5) [inline app/src/main/java/org/commcare/dalvik/abha/ui/main/fragment/AbhaValidationFragment.kt]: unused ?

File: app/src/main/res/raw/ccind.cer [hunk 3]
```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```
Comment (shubham1g5) [inline app/src/main/res/raw/ccind.cer]: @akashkj Is this a certificate specific to Dimagi ? Does this expire ever ?

**Review by shubham1g5 — COMMENTED** [review]
Looks good though I am mostly relying on user testing by ID in terms of verifying the functionality works correctly.

File: app/src/main/res/raw/ccind.cer (hunk 3 above)
Comment (akashkj) [reply app/src/main/res/raw/ccind.cer]: This certificate seem to be of custom domain where an instance of HQ is hosted currently. It's expiry is of 3 months with auto renewal from letsencrypt.
@RawatH Why is this certificate added, can you please tell?

File: app/src/main/java/org/commcare/dalvik/abha/ui/main/fragment/AbhaValidationFragment.kt (hunk 2 above)
Comment (RawatH) [reply app/src/main/java/org/commcare/dalvik/abha/ui/main/fragment/AbhaValidationFragment.kt]: Removed

File: app/src/main/res/raw/ccind.cer (hunk 3 above)
Comment (RawatH) [reply app/src/main/res/raw/ccind.cer]: Removed

File: .idea/other.xml (hunk 1 above)
Comment (RawatH) [reply .idea/other.xml:1]: Will update in upcoming PR

**Review by akashkj — APPROVED** [review]

File: app/src/main/java/org/commcare/dalvik/abha/ui/main/activity/TestActivity.kt [hunk 4]
```diff
@@ -54,7 +54,7 @@ class TestActivity : AppCompatActivity() {
         val intent = Intent(action).apply {
             putExtras(
                 bundleOf(
-                    "mobile_number" to "",
+                    "mobile_number" to "9560833229",
```
Comment (shubham1g5) [inline app/src/main/java/org/commcare/dalvik/abha/ui/main/activity/TestActivity.kt:63]: is this a real number ? If so, who does it belong to ?

Comment (RawatH) [reply app/src/main/java/org/commcare/dalvik/abha/ui/main/activity/TestActivity.kt:63]: The TestActivity is for testing purpose. This activity basically used to launch and the test the all possible flows. We can say its acting as a proxy for CommCare APP.

The phone number belongs to me and its used to receive the OTP for authentication via ABDM api.

File: app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -5,6 +5,11 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
+    <uses-feature android:name="android.hardware.camera" />
+    <uses-feature android:name="android.hardware.camera.autofocus"/>
+
+    <uses-permission android:name="android.permission.CAMERA"/>
```
Comment (shubham1g5) [inline app/src/main/AndroidManifest.xml:11]: does the code-scanner do the runtime permission manangement for this permission ?

**Review by shubham1g5 — APPROVED** [review]

File: app/src/main/AndroidManifest.xml (hunk 5 above)
Comment (RawatH) [reply app/src/main/AndroidManifest.xml:11]: Yes ,for API level 23 I will be adding the runtime permission. Working on it.

**Pull request merged**